<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KPlato::GanttChartDisplayOptions</class>
 <widget class="QWidget" name="KPlato::GanttChartDisplayOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>515</height>
   </rect>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Timeline</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QRadioButton" name="ui_timeLineHide">
          <property name="toolTip">
           <string>Do not show the timeline</string>
          </property>
          <property name="text">
           <string>Hide</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="ui_timeLineForeground">
          <property name="toolTip">
           <string>Show timeline in the foreground</string>
          </property>
          <property name="text">
           <string>Foreground</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="ui_timeLineBackground">
          <property name="toolTip">
           <string>Show timeline in the background</string>
          </property>
          <property name="text">
           <string>Background</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="toolTip">
             <string>Update interval in minutes, 0 disables update</string>
            </property>
            <property name="text">
             <string>Interval:</string>
            </property>
            <property name="buddy">
             <cstring>ui_timeLineInterval</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ui_timeLineInterval">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCheckBox" name="ui_timeLineUseCustom">
          <property name="text">
           <string>Use custom line</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="toolTip">
             <string>Line width</string>
            </property>
            <property name="text">
             <string>Stroke:</string>
            </property>
            <property name="buddy">
             <cstring>ui_timeLineStroke</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ui_timeLineStroke">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="toolTip">
             <string>Line color</string>
            </property>
            <property name="text">
             <string>Color:</string>
            </property>
            <property name="buddy">
             <cstring>ui_timeLineColor</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KColorCombo" name="ui_timeLineColor">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Decorations</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="ui_showRowSeparators">
        <property name="text">
         <string>Show Row Separators</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Freedays:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="freedays">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Additional text information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="ui_showTaskName">
        <property name="text">
         <string>Task name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="ui_showResourceNames">
        <property name="text">
         <string>Resource names</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Additional graphics information</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QCheckBox" name="ui_showCriticalPath">
        <property name="text">
         <string>Critical path</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="ui_showDependencies">
        <property name="text">
         <string>Task dependencies</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="ui_showNegativeFloat">
        <property name="text">
         <string>Negative float</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="ui_showCriticalTasks">
        <property name="text">
         <string>Critical tasks</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="ui_showCompletion">
        <property name="text">
         <string>Task completion</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="ui_showPositiveFloat">
        <property name="text">
         <string>Positive float</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="ui_showSchedulingError">
        <property name="text">
         <string>Scheduling errors</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="ui_showTimeConstraint">
        <property name="text">
         <string>Time constraints</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
